--- a/eurasia_km/services4/srvkm/env/linux/osfunc.c
+++ b/eurasia_km/services4/srvkm/env/linux/osfunc.c
@@ -3700,7 +3700,7 @@ PVRSRV_ERROR OSAcquirePhysPageAddr(IMG_V
     bMMapSemHeld = IMG_TRUE;
 
     /* Get page list */
-    psInfo->iNumPagesMapped = get_user_pages_remote(current, current->mm, uStartAddr, psInfo->iNumPages, FOLL_WRITE,0, psInfo->ppsPages, NULL);
+    psInfo->iNumPagesMapped = get_user_pages_remote(current, current->mm, uStartAddr, psInfo->iNumPages, FOLL_WRITE, psInfo->ppsPages, NULL);
 
     if (psInfo->iNumPagesMapped >= 0)
     {
